docs/configuration.md

```diff
@@ -139,6 +139,7 @@ storage:
     multipartcopythresholdsize: 33554432
     rootdirectory: /s3/object/name/prefix
     usedualstack: false
+    loglevel: logdebug
```

Comment:
I would prefer if this were just `debug`, not `logdebug` 🤔 

AWS SDK `LogLevelType` defines the log level as an `int` so there is nothing preventing from defining our _string representations_ of the different levels that don't stutter

Reply:
Agree, done.